src/stylesheets/components/key-factors.scss

```diff
@@ -0,0 +1,110 @@
+/*----------- Key Factors -----------*/
+
+.key-factors {
+  // background-image: url('../images/body.svg');
+  // background-position: center;
+  // background-repeat: no-repeat;
+  color: $florence-gray;
+  margin-top: 150px;
+
+  @media (max-width: $screen-sm) {
+    margin-top: 100px;
+  }
+
+  h3 {
+    color: $florence-orange;
+    font-size: 1em;
+    margin-bottom: -8px;
+    text-transform: uppercase;
+  }
+}
+
+.section-title {
+  color: $florence-blue;
+  font-size: 0.65em;
+  letter-spacing: 2px;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.section-intro {
+  color: $florence-blue;
+  font-family: $secondary-font;
+  font-size: 1.7em;
+  line-height: 1.6em;
+  margin: auto;
+  text-align: center;
+  width: 57vw;
+
+  @media (max-width: $screen-sm) {
+    font-size: 1.4em;
+    width: 85vw;
+  }
+}
+
+.factor-bg {
+  background-image: url('../images/body.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.factor-box {
+  background-color: $white;
+  border-radius: 9px;
+  font-size: 0.7em;
+  line-height: 1.4em;
+  overflow-x: visible;
+  padding: 25px;
+  position: relative;
+  width: 250px;
+
+
+  &:before {
```

Comment:
while in the past `:` has been acceptable for both pseudo-elements and pseudo-classes, spec now insists on `::` for the former and `:` for the latter (compare the spec write-ups https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements and https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes). this won't break anything (yet……), but it's good to get in the habit of `::before` and `::after`